router/pods_test.go

```diff
@@ -800,30 +800,26 @@ func TestGetHealthCheckFromPodPort(t *testing.T) {
 
 // TestValidatePath tests the internal function to validate a proper path used in annotations
 func TestValidatePath(t *testing.T) {
-	testNoPrefix := "test"
-	testPathFail := "/test/%2a/%"
-	testPathPass := "/test/%2a/aa/a"
-	testPathPassProperEncoding := "/test/par%2ate/aa/a"
-	testPathFailInvalidEncoding := "/test/hello%zzworld/aa/a"
```

Comment:
Sure, but the some users (Martin) want to use a regex for matching a path. How would a URI validator make any sense of that? The input to nginx and our path is not a URI it's a path match pattern (or at least this PR makes it so).